play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/EvaluationSettings.java

```diff
@@ -48,7 +48,7 @@ final class EvaluationSettings {
 
     private long ambientLimitBytes = MemoryFootprint.toBytes(10);
 
-    private long activeLimitBytes = MemoryFootprint.toBytes(10);
+    private long activeLimitBytes = MemoryFootprint.toBytes(100);
```

Comment:
```suggestion
    private long activeLimitBytes = MemoryFootprint.toBytes(/* megaBytes = */100);
```

I found this confusing as expressed

Reply:
super nit: I guess it's really mebiByte ?